test/Fondue.test.js

```diff
@@ -6,8 +6,10 @@ const otfFont = async () => {
 	);
 };
 
-const ttfFont = async () => {
-	return await loadFondue("./test/fixtures/letterA/letterA.ttf");
+const WhatCanThisFontDo = async () => {
```

Comment:
This hurts readability of the tests a bit IMO. All the fixture helpers refer to the kind of font, which is usually important for the tests. `WhatCanThisFontDo` doesn't say anything about the kind of font?

Reply:
The idea was to put several testable things inside this font, ranging from metadata to glyph count, and in the future OT features, etc. So I'd like to emphasize it's a versatile test font. Perhaps simply `WFTestFont`? The readme contains details about exactly what this font does and how it can be used for testing?

I'll change it to that for now (agreed that `WhatCanThisFontDo` is more cute than informative) and we can improve it during future changes.